config/postcss/media.js

```diff
@@ -0,0 +1,20 @@
+/* tslint:disable object-literal-sort-keys */
```

Comment:
tslint is deprecated and we shouldn't be using it right here. Or typescript for that matter :)

Reply:
Removed all such comments